dcpy/lifecycle/package/generate_metadata_assets.py

```diff
@@ -0,0 +1,29 @@
+from dcpy.models.product.dataset.metadata import Metadata
+import argparse
+from pathlib import Path
+from jinja2 import Environment, FileSystemLoader
+import typer
+
+app = typer.Typer()
+
+
+@app.command()
+def generate_metadata_assets(yaml_file_path, html_template_path, output_html_path):
```

Comment:
I wood add types in fn signature here and in the remaining functions. 

For example, this fn signature would look like this: 

```python
def generate_metadata_assets(yaml_file_path: str, html_template_path: str, output_html_path: str) -> Path:
...
```

Reply:
Quick question: For the get_metadata function that returns the SocrataMetada, do I just say object? Do I need to add the SocrataMetada class into the module?